bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           All nodes should be running an ovs pod, {{"{{"}} $labels.node {{"}}"}} is not.
       expr: |
         (kube_node_info unless on(node) kube_pod_info{namespace="openshift-sdn",  pod=~"ovs.*"}) > 0
-      for: 20m
+      for: 10m
```

Comment:
why change the times here?

Reply:
Just general tightening up of alerts. 20 minutes of proxy staleness would be insane.

Reply:
Agreed, 20 minutes is a bit too much.